### DISCUSSION TITLE
Update plugin for using new features in WP

### DISCUSSION THREAD

**Discussion by seredniy** [spatie/wordpress-ray#46] [Q&A] [answered]
I am trying to use the new feature of Ray `ray($var)->label('info');` with WordPress plugin, but ... Ray version in WordPress Plugins is too old. Please, update it, when you will have time.

**Comment by freekmurze** [comment, marked as answer]
I've tagged a new release 1.5.0, which contains the latest feature of spatie/ray. Enjoy!

**Reply by seredniy** [reply]
Thank you!
